ExpenseSplit/HomeScreen.swift

```diff
@@ -0,0 +1,162 @@
+//
+//  HomeScreen.swift
+//  ExpenseSplit
+//
+//  Created by Ankita Katoch on 8/9/24.
+//
+
+import Foundation
+import SwiftUI
+
+struct Friend: Identifiable {
+    let id = UUID()
+    let name: String
+    let avatar: String
+    let balance: String
+    let isOwed: Bool
+}
```

Comment:
Your Friend model looks solid! I have a suggestion that could make it even more flexible and easier to work with. Since properties like balance and isOwed might change over time, it could be beneficial to make them variables (var). Additionally, you might want to consider adding some computed properties to encapsulate logic related to balance and isOwed. For example, a computed property could automatically determine if a friend’s balance is zero and return a ‘settled up’ status. This approach could help simplify and centralize your logic, making the model more dynamic and adaptable. Just a thought—keep up the great work!